scripts/src/main/resources/scripts/command/project

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+
+# $1: project to setup (omit for all all projects)
+function doSetup() {
+  if [ -n "${1}" ]
+  then
+    local project_file="${1}"
+    if [ ! -f "${project_file}" ]
+    then
+      project_file="${SETTINGS_PATH}/projects/${1}.properties"
+    fi
+    if [ ! -f "${project_file}" ]
+    then
+      doFail "Could not find ${project_file}"
+    fi
+    doImportProject "${project_file}" "force"
+  elif [ -d "${SETTINGS_PATH}/projects" ]
+  then    
+    local last_cwd file project_force
+    project_force=""
+    if doIsForce
+    then
+      project_force="force"      
+    fi
+    last_cwd="${PWD}"
+    for file in "${SETTINGS_PATH}"/projects/*.properties
+    do
+      doImportProject "${file}" "${project_force}"
+    done
+    cd "${last_cwd}" || exit 255
+  fi
+}
+
+# $1: path to properties file
+function doImportProject() {
+  if [ ! -f "${1}" ]
+  then
+    return
+  fi
+  project_path=""
+  project_workingsets=""
+  project_workspace="main"
+  project_git_url=""
+  project_git_branch=""
+  project_build_path="."
+  project_eclipse="import"
```

Comment:
I was wondering about this default by the way. It was making much trouble for us finding out how to deactivate eclipse import without reading the code. Even as this feature is not reliably working. I would suggest to remove the default = import.

Anyhow if you like to have a separate pr for that I will go for it.
Obviously, this will cause a changed behavior for ide upgrades.

Reply:
Default changed as requested.